CHANGELOG.md

```diff
@@ -34,8 +36,25 @@ and includes an additional section for migration notes.
 
 ### Migration Notes
 
+- Create a new bucket `PREFIX-orca-reports` in the same account and region as your primary orca bucket.
+  - Give the bucket a [lifecycle configuration](https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-set-lifecycle-configuration-intro.html) with an expiration period of 30 days.
+  - Follow instructions in https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-s3-bucket/ to set up permission policy.
+  - Modify the permissions for your primary Orca bucket.
+    - Under the `Cross Account Access` policy, add `s3:GetInventoryConfiguration` and `s3:PutInventoryConfiguration` to Actions.
+  - Update the `buckets` variable in your `tfvars` file by adding the bucket.
+    ```
+    buckets = {
+      ...
+      orca_reports = {
+        name = "PREFIX-orca-reports"
+        type = "orca-reports"
+      },
+      ...
+    }
```

Comment:
This sets up our iam permissions. I could try to set those manually if desired, or we could wait until we reevaluate how we handle buckets.

Reply:
Added specific permissions for reports buckets. Buckets still required in buckets list.

Reply:
It is good that this has been broken out but not exactly what I was thinking.  For this first round through I think we are okay but, I would really like to see this pulled out into its own IAM role specific for the internal recovery lambdas and get away from one massive IAM role that we are applying to all lambdas to help us have a better security posture. This would dove tail into the work Nate is doing on some of our legacy tasks. Another thing to keep in mind is instead of adding it to the `buckets` variable, you should be able to reuse the `orca_reports_bucket_arn` variable here for the pathing.

Reply:
Removed report bucket from bucket list.

Reply:
Permissions work exists at https://bugs.earthdata.nasa.gov/browse/ORCA-387